### DISCUSSION TITLE
Case insensitive when searching for user with user attributes

### DISCUSSION THREAD

**Discussion by Aleson95** [keycloak/keycloak#13714] [Q&A] [answered]
Is there anyway we can search case insensitive when searching for user with user attributes?
I tried the [api](https://www.keycloak.org/docs-api/15.1/rest-api/index.html#_users_resource) here but it query the user attribute by case sensitive

**Comment by hmlnarik** [comment, marked as answer]
Not in the legacy store, but we have plans to address this in the new store, see #10245
